bigtop-packages/src/charm/spark/layer-spark/actions/restart-spark-job-history-server

```diff
@@ -15,22 +15,22 @@
 # limitations under the License.
 import sys
 
+from charmhelpers.core import hookenv, host
+from charms.reactive import is_state
 
-try:
-    from charmhelpers.core import host, hookenv, unitdata
-    from jujubigdata import utils
-    charm_ready = True
-except ImportError:
-    charm_ready = False
 
-if not charm_ready:
-    from subprocess import call
-    call(['action-fail', 'Spark service not yet ready'])
-    sys.exit(1)
```

Comment:
This was originally intended to handle the case where the action somehow ran before the bootstrap code, but I don't think that's actually possible, so +1 to this change.